docker-compose.cypress.yml

```
@@ -7,3 +7,10 @@ services:
     user: ${DOCKER_UID}:${DOCKER_GID}
     volumes:
       - ./e2e:/e2e
+
+  node:
+    image: circleci/node:14
+    working_dir: /learning-analytics-playground
+    user: ${DOCKER_UID}:${DOCKER_GID}
+    volumes:
+      - .:/learning-analytics-playground
```

We often use `/app` instead.

```suggestion
    working_dir: /app
    user: ${DOCKER_UID}:${DOCKER_GID}
    volumes:
      - .:/app
```